src/menu/node-menu.component.ts

```diff
@@ -52,7 +54,7 @@ export class NodeMenuComponent implements OnInit, OnDestroy {
 
   public ngOnInit(): void {
     this.disposersForGlobalListeners.push(this.renderer.listenGlobal('document', 'keyup', this.closeMenu.bind(this)));
-    this.disposersForGlobalListeners.push(this.renderer.listenGlobal('document', 'click', this.closeMenu.bind(this)));
+    this.disposersForGlobalListeners.push(this.renderer.listenGlobal('document', 'mousedown', this.closeMenu.bind(this)));
```

Comment:
Because opening of a left menu is an left mouse event - click event, and if you add event listener for closing menu on a click (when a menu is initializing), it fires right after opening which cause immediate closing.